config/v1/types_cluster_version.go

```diff
@@ -50,6 +50,10 @@ type ClusterVersionSpec struct {
 	// +optional
 	DesiredUpdate *Update `json:"desiredUpdate,omitempty"`
 
+	// automaticUpdates enables automatic updates.
+	// +optional
+	AutomaticUpdates URL `json:"automaticUpdates,omitempty"`
```

Comment:
Why is this a URL..?

Also the design for auto-updates should aim to be enabled for all unless specifically specified. And needs to bring in existing users where this field was not set.

Reply:
Unfortunately we can’t turn on auto updates for existing users.  We’d have to get them to turn it on themseves

Reply:
Fixed to be a boolean with d3cfead4 -> 0b9690b5.